docs/usage.rst

```diff
@@ -4,32 +4,40 @@
 Usage
 =====
 
-Create a :class:`~parsel.selector.Selector` object for the HTML or XML text
-that you want to parse::
+Create a :class:`~parsel.selector.Selector` object for the input text that you want to parse and use.
 
-    >>> from parsel import Selector
-    >>> text = "<html><body><h1>Hello, Parsel!</h1></body></html>"
-    >>> selector = Selector(text=text)
-
-Then use `CSS`_ or `XPath`_ expressions to select elements::
+For HTML or XML, use `CSS`_ or `XPath`_ expressions to select elements::
```

Comment:
I know this comes from the original string, but I think it was not technically accurate (e.g. text is not an element).

```suggestion
For HTML or XML, use `CSS`_ or `XPath`_ expressions to select data::
```

Reply:
I guess if you don't use get/getall we could say the selector is handling elements. But overall its true we generally just want the data inside those elements anyway.